src/components/UserProfile/VolunteeringTimeTab/VolunteeringTimeTab.jsx

```diff
@@ -85,17 +88,40 @@ const WeeklyCommitedHours = props => {
   if (!props.canEdit) {
     return <p>{props.userProfile.weeklyComittedHours}</p>;
   }
+  const handleChange = e => {
+    // Maximum and minimum constants on lines 9 & 10
+    // Convert value from string into easy number variable
+    const value = parseInt(e.target.value);
+    if (value > MAXIMUM_WEEK_HOURS) {
+      // Check if Value is greater than total hours in one week
+      alert(`You can't commit more than ${MAXIMUM_WEEK_HOURS} hours per week.`);
+      if (value === 169) {
```

Comment:
Why are you checking this? if value === 169 and setting maximum committed time to 168. 
Either you should do this for every value that is greater than 168 or don't do it.

What's the purpose of this ( please let me know If I am missing anything that I need to know )

Reply:
Great point! I originally was doing something similar to what you suggested but I wanted the alerts to also pop up when users are using their arrow keys as well. I set this conditional to check if the value is 169 because otherwise, I want it to stay the same as what the user is attempting to enter. If I have it check for any value greater than 168 then if a user attempts to write 40 but accidentally types 400 their value will be set to 168. I only want the value to be reset to 168 if it's 169. This way users who are using arrow keys to go up or down will hit the alerts and have their values set to the minimums and maximums, while users who are typing in their values don't have to go through the trouble of retyping if they make a mistake. Let me know if you have any further questions or suggestions on how this can be done better!